public/app/core/utils/CorsSharedWorker.ts

```diff
@@ -0,0 +1,52 @@
+// Almost identical to CorsWorker.ts. Main difference being it allows loading a SharedWorker if browser supports it
+
+export function sharedWorkersSupported() {
+  return typeof window.SharedWorker !== 'undefined';
+}
+
+class SharedWorkerNotSupported implements SharedWorker {
+  onerror() {}
+  // @ts-ignore
```

Comment:
Might lead to runtime issue as it does not have value

Reply:
As mentioned above, will try with `class extends null` when worker is not supported